assets/js/editor-new.js

```diff
@@ -310,10 +310,20 @@
   const uploadImage = function(file, onSuccess, onError) {
     const galleryUrl = '/api/galeries/' + document.body.getAttribute('data-gallery') + '/images/'
 
+    if (file.type.indexOf("image") !== 0) {
+      onError("L'image '" + file.name + "' a un format invalide !")
+      return
+    }
+
+    const filesize = (file.size / 1024)  // KB
```

Comment:
Pour éviter d'avoir plein de chiffres après la virgule
```suggestion
    const filesize = Math.round(file.size / 1024) // KB
```